README.md

```diff
@@ -233,15 +233,19 @@ Type: Integer
 Default: None
 
 Specifies the number of free IP addresses that the `ipamD` daemon should attempt to keep available for pod assignment on the node.
-For example, if `WARM_IP_TARGET` is set to 10, then `ipamD` attempts to keep 10 free IP addresses available at all times. If the
+For example, if `WARM_IP_TARGET` is set to 5, then `ipamD` attempts to keep 5 free IP addresses available at all times. If the
 elastic network interfaces on the node are unable to provide these free addresses, `ipamD` attempts to allocate more interfaces
 until `WARM_IP_TARGET` free IP addresses are available.
+
+**NOTE!** Avoid this setting for large clusters, or if the cluster has high pod churn. Setting it will cause additional calls to the
+EC2 API and that might cause throttling of the requests. It is strongly suggested to set `MINIMUM_IP_TARGET` when using `MINIMUM_IP_TARGET`.
```

Comment:
"It is strongly suggested to set `MINIMUM_IP_TARGET` when using `MINIMUM_IP_TARGET`" doesn't make sense to me. I think you mean to say "It is strongly suggested to set `MINIMUM_IP_TARGET` instead"?

Reply:
Or perhaps you mean "It is strongly suggested to set `MINIMUM_IP_TARGET` when using `WARM_IP_TARGET`"?